test/intl402/DateTimeFormat/prototype/format/timedatestyle-en.js

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-date-time-style-pattern
+description: Checks basic handling of timeStyle and dateStyle.
+features: [Intl.DateTimeFormat-datetimestyle]
+locale: [en-US]
+---*/
+
+const date = new Date("1886-05-01T14:30:17");
```

Comment:
This need to be "1886-05-01T14:12:47Z"
14:30:17 does not make sense since you are expecting "2:12:47 PM" below.
Also, according to https://ecma-international.org/ecma-262/#sec-date-time-string-format
You need he ending Z to denote "for UTC"